drake/examples/Cars/simulateLCM.cpp

```diff
@@ -118,15 +139,20 @@ int main(int argc, char* argv[]) {
 
     for (int actuator_idx = 0; actuator_idx < tree->actuators.size();
          actuator_idx++) {
-      if (strcmp(tree->actuators[actuator_idx].name.c_str(),
-                 "steering_angle") == 0) {
+
+      const std::string & actuatorName = tree->actuators[actuator_idx].name;
+
+      if (strcmp(actuatorName.c_str(), "steering_angle") == 0 ||
```

Comment:
you could rename the urdf actuator to match the (implied) sdf actuator name.  that might be cleaner.